openedx/frontend-component-footer
# fix: variable name for the accessibility link

This PR changes the env variable that is required for to show the accessibility page link in the footer. The variable name needed to change because it was not found in the course authoring env config. All the env variable values for the footer are inherited from its parent.

Testing
1. Add a local version of this repo to your `frontend-app-course-authoring/module.config.js` file
```
{ moduleName: '@edx/frontend-component-footer', dir: '../frontend-component-footer', dist: 'dist' },
```
2. Navigate to a page in studio that uses the `frontend-app-course-authoring`
3. Should not see "Accessibility Accommodation Request" in the footer
4. In `frontend-app-course-authoring/env.dev`, update `ENABLE_ACCESSIBILITY_PAGE` and restart npm
5. Reload MFE page
6. Should see "Accessibility Accommodation Request" in the footer
